sqlglot/parser.py

```diff
@@ -2134,7 +2135,7 @@ def _parse_cast(self, strict):
 
         return self.expression(exp.Cast if strict else exp.TryCast, this=this, to=to)
 
-    def _parse_convert(self):
+    def _parse_convert(self, strict):
```

Comment:
strict is never used here, you should add the try convert logic in base

Reply:
The function in the base class is now also using the strict parameter and will transpile to a TryCast or Cast.